packages/scan/src/extrinsic/treasury/tip.js

```diff
@@ -74,8 +76,26 @@ async function saveTipTimeline(hash, state, args, indexer, sort) {
     hash,
     args,
     state,
-    meta: meta.toJSON(),
+    meta,
   });
+
+  const medianValue = computeTipValue((meta && meta.tips) || []);
+
+  const tipCol = await getTipCollection();
+  await tipCol.updateOne(
+    { hash },
+    {
+      $set: {
+        medianValue,
+        meta,
+        state: {
+          indexer,
+          state,
+          args,
+        },
+      },
+    }
+  );
```

Comment:
Don't do tip update in this `saveTipTimeline` function. Better to extract another function for clean code.